[Term Entry] Pandas DataFrame: .sort_values() (#1601)
Codecademy/docs: content/pandas/concepts/dataframe/terms/sortvalues/sortvalues.md

---
Title: '.sort_values()'

> Description: 'Sorts values in a DataFrame based on the specified column or columns.'

Author: I think this option would be more descriptive, but I can use the suggested change, too.
```suggestion
Description: 'Sorts values in a DataFrame by the specified column or row.'
```

> Description: 'Sorts values in a DataFrame based on the specified column or columns.'

Reviewer: ```suggestion
Description: 'Sorts values in a DataFrame based on a selection of columns or rows.'
```

> Description: 'Sorts values in a DataFrame based on the specified column or columns.'

Reviewer: @alespros My edit is not as concise as I'd like, but I think it's important to detail that the sorting is not limited to one label in either direction.

Subjects:
  - 'Computer Science'
  - 'Data Science'
Tags:
  - 'Data Structures'
  - 'Pandas'
CatalogContent:
  - 'learn-python-3'
  - 'paths/data-science'
---

The **`.sort_values()`** function sorts values in a [`DataFrame`](https://www.codecademy.com/resources/docs/pandas/dataframe) along the selected axis and returns a [`DataFrame`](https://www.codecademy.com/resources/docs/pandas/dataframe) with sorted values or None.

## Syntax

```pseudo
dataframevalue.sort_values(by, *, axis=0, ascending=True, inplace=False, kind='quicksort', na_position='last', ignore_index=False, key=None)
```

`.sort_values()` uses the following parameters:

> - `by`: `Name` or `list of names` to sort by.

Author: ```suggestion
- `by`: A label or list of labels to sort by (levels can also specified if applicable)..
```
